docs/cep/getting-started-stream-workers.md

```diff
@@ -0,0 +1,102 @@
+---
+sidebar_position: 10
+title: Getting Started with Stream Workers
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+This page explains how to create a stream worker that reads data added to a collection and sends the data to a stream.
+
+## Create a Stream Worker
+
+Refer to the [Stream Workers](/cep/index.md) section for an explanation of how stream workers function.
+
+1. [Log in to your Macrometa account](https://auth-play.macrometa.io/).
+1. Click **Stream Worker**.
+1. In the Editor tab, you must define the stream worker. Copy and paste the following code block in the code editor on the Editor tab.
+
+```sql
```

Comment:
Take a look at the numbering before and after the code block in your build. Right now (on mine, anyway), the numbering is restarting after the code block. You can fix that by indenting the whole code block one tab. VSC is clever enough to allow you to do it all at once, just select the whole block and then tab.